code/drasil-website/lib/Drasil/Website/About.hs

```diff
@@ -99,6 +100,10 @@ aboutParagraph8 paperGOOL = S "As described in the" +:+ namedRef paperGOOL (S "G
   \generation, allowing Drasil to more efficiently generate code in several languages, including Python, Java, C-Sharp, and C++."
 
 -- | Paragraph providing a link to Drasil papers and documents
-aboutParagraph9 :: Reference -> Sentence
-aboutParagraph9 papersWiki = S "A list of papers and documents written about Drasil can be found in the" +:+ namedRef papersWiki (S "Drasil Papers \
-  \and Documents") +:+ S "wiki page"
\ No newline at end of file
+aboutParagraph9 :: Reference -> Reference -> Reference -> Reference -> Sentence
+aboutParagraph9 papersWiki icsePositionPaper danPoster wellUnderstoodPaper = 
+  S "A list of papers and documents written about Drasil can be found in the" +:+ 
+  namedRef papersWiki (S "Drasil Papers and Documents") +:+ S "wiki page. In particular, there is a" +:+ 
+  namedRef icsePositionPaper (S "Old Position Paper") +:+ S "outlining our original ideas, a" +:+ 
+  namedRef danPoster (S "Drasil Poster") +:+ S ", and a" +:+ 
+  namedRef wellUnderstoodPaper (S "Well-Understood Paper") +:+ S "discussing key concepts."
```

Comment:
I would make the following changes:
1. use "an", since the next word is "old", which starts with a vowel sound
2. remove the repeated articles (i.e., the other "a/an"s) to improve readability (especially for change 4 below)
3. use `+:+.` instead of `+:+` before change 4 below, since the final period "splits off" quite nicely
4. use `foldlList` to format inline lists of items in a consistent way

```suggestion
  namedRef papersWiki (S "Drasil Papers and Documents") +:+ S "wiki page. In particular, there is an" +:+.
  foldlList Comma List [
     namedRef icsePositionPaper (S "Old Position Paper") +:+ S "outlining our original ideas",
     namedRef danPoster (S "Drasil Poster"),
     namedRef wellUnderstoodPaper (S "Well-Understood Paper") +:+ S "discussing key concepts"
  ]
```

Hopefully these changes make sense; I haven't actually run this code, so make sure to test it and make sure that the output is as expected!